.circleci/config.yml

```diff
@@ -18,6 +18,18 @@ commands:
           source test_env/bin/activate
           cd "featuretools-$(python setup.py --version)/"
           pytest
+  install-ft-complete-and-run-import-tests:
+    description: "run unit tests on packaged testing files"
+    steps:
+      - run: |
+          virtualenv test_ft_complete_env
+          source test_ft_complete_env/bin/activate
+          python setup.py sdist
+          FEATURETOOLS_VERSION=$(python setup.py --version)
+          tar -zxvf "dist/featuretools-${FEATURETOOLS_VERSION}.tar.gz"
+          pip install -e "featuretools-${FEATURETOOLS_VERSION}/[complete]"
+          python -c "import featuretools_tsfresh_primitives"
```

Comment:
Can we also test importing `featuretools.tsfresh`

Reply:
Fixed.